docs/tutorials/javascript.rst

```diff
@@ -0,0 +1,130 @@
+.. _tutorial-javascript:
+
+Kinto for JavaScript applications
+#################################
+
+Kinto is particularly relevant for the Web, thus it is very probable that you'll
+want to interact with the server using JavaScript.
+
+This tutorial presents the basics, but you can look at
+:ref:`the applications examples <app-examples>`, most of them use JavaScript.
+
+
+Offline and synchronization
+---------------------------
+
+We built :github:`kinto.js <Kinto/kinto.js>`, a JavaScript client dedicated
+to synchronizing remote records with a local database.
+
+The best way to start is :rtd:`to follow the dedicated tutorial <kintojs>` !
```

Comment:
I thought also, but then realize there were inconvenients: 

* the global index is bypassed (and can give some info to get started)
* the link has to be maintained and can be broken more easily